### PR TITLE
[WIP] French translation

### PR DESCRIPTION
**Pull request by Poussinou**
Hi,

Here is the French translation, made with [Stringlate](https://github.com/LonamiWebs/Stringlate).

Some translations might be too long to fit the labels on the screen in the app, Can you include this translation, then compile and report me if everything is ok, before pushing? :)

### CODE REVIEW

**Comment by aaronjwood** [comment]
Thanks for this! I'll test it out ASAP and let you know about the formatting.

**Comment by aaronjwood** [comment]
Just a couple of issues that I found:

screenshot_20180109-202338

screenshot_20180109-202358

**Comment by Poussinou** [comment]
It should be ok now

File: app/src/main/res/values-fr/array.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <item>50</item>
+        <item>25</item>
+        <item>A</item>
+    </string-array>
+    <string-array name="upperDrawer">
+        <item>\u2601 Scanner l\'hôte WAN</item>
+        <item>\u2194 Recherche DNS</item>
+    </string-array>
+    <string-array name="lowerDrawer">
+        <item>\u2699 Paramètres</item>
```
Comment (aaronjwood) [inline app/src/main/res/values-fr/array.xml]: This looks a bit off compared to the original (both text and unicode symbols)
```
    <string-array name="lowerDrawer">
        <item>\u2195 Update OUI Database</item>
        <item>\u2195 Update Port Database</item>
        <item>\u2699 Settings</item>
    </string-array>
```

**Comment by Poussinou** [comment]
My apologizes, Stringlate (or me obviously) had some problems with the array file... Sorry for the loss of time...

I'll open an issue, with a link to this PR ;)

**Comment by aaronjwood** [comment]
No problem, thanks for opening the issue! I'll test this ASAP. I'm in the middle of relocating across the country right now so it might take me a bit :)

**Comment by Poussinou** [comment]
No problem :)

**Pull request merged**